Resources/config/core.xml

```diff
@@ -66,7 +66,7 @@
         </service>
         <!-- twig -->
         <service id="sonata.admin.twig.global" class="Sonata\AdminBundle\Twig\GlobalVariables">
-            <argument type="service" id="service_container"/>
+            <argument type="service" id="sonata.admin.pool"/>
```

Comment:
Forget about my concern. If you extend this, you should have created a new service id for it.